portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/displaypagetemplate/DisplayPageTemplatesWithItemPreview.testcase

```diff
@@ -480,4 +480,116 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-131040. Only the public vocabularies are available in preview item selector."
+    @priority = "4"
+    test OnlyPublicVocabulariesPreviewSelector {
+        task ("Add a category in Topic, Audience and Stage") {
+         	JSONCategory.addCategory(
+            	groupName = "Global",
+                title = "Topic Category Name",
+                vocabularyName = "Topic");
+
+         	JSONCategory.addCategory(
+         	    groupName = "Global",
+            	title = "Audience Category Name",
+            	vocabularyName = "Audience");
+
+        	JSONCategory.addCategory(
+             	groupName = "Global",
+             	title = "Stage Category Name",
+             	vocabularyName = "Stage");
+        }
+
+        task ("Add a display page template for Category") {
+        	JSONLayoutpagetemplate.addDisplayPageTemplateEntry(
+           	 	contentType = "Category",
+          	    displayPageTemplateEntryName = "Display Page Name",
+            	groupName = "Test Site Name");
+        }
+
+        task ("Add a Button fragment to display Page Template") {
+            DisplayPageTemplatesAdmin.openDisplayPagesAdmin(siteURLKey = "test-site-name");
+
+            DisplayPageTemplatesAdmin.gotoDisplayPage(displayPageName = "Display Page Name");
+
+            PageEditor.addFragment(
+             	collectionName = "Basic Components",
+             	fragmentName = "Button");
+        }
+
+        task ("Mapping the Name field to Heading fragment") {
+         	PageEditorMapping.mapEditableTextToField(
+             	field = "Vocabulary",
+             	fragmentName = "Button",
+            	id = "link");
+        }
+  
+		task ("Select the category via preview item selector") {
```

Comment:
Modify task message to align with steps. Like access to Preview With item selector